Update Readme.md (#1)
DevExpress-Examples/asp-net-web-forms-scheduler-bind-to-oracle: Readme.md

<!-- default badges list -->
[![](https://img.shields.io/badge/Open_in_DevExpress_Support_Center-FF7200?style=flat-square&logo=DevExpress&logoColor=white)](https://supportcenter.devexpress.com/ticket/details/T223139)
[![](https://img.shields.io/badge/📖_How_to_use_DevExpress_Examples-e9f6fc?style=flat-square)](https://docs.devexpress.com/GeneralInformation/403183)
<!-- default badges end -->

> # Scheduler for ASP.NET Web Forms - How to bind the scheduler to the Oracle data base using the ObjectDataSource component

Reviewer: ```suggestion
# Scheduler for ASP.NET Web Forms - How to bind the scheduler to the Oracle database using the ObjectDataSource component
```

> This example demonstrates how to use facilities of the **System.Data.OracleClient** assembly in the `ObjectDataSource` component to bind  [ASPxScheduler](https://docs.devexpress.com/AspNet/3830/components/scheduler/visual-elements/aspxscheduler) to the Oracle data base and perform CRUD operations. 

Reviewer: ```suggestion
This example demonstrates how to use **System.Data.OracleClient** assembly facilities in the `ObjectDataSource` component to bind [ASPxScheduler](https://docs.devexpress.com/AspNet/3830/components/scheduler/visual-elements/aspxscheduler) to the Oracle database and perform CRUD operations. 
```

> For implementation details, refer to the following article: [How to bind ASPxScheduler to the Oracle data base using the ObjectDataSource component (description)](https://supportcenter.devexpress.com/ticket/details/t223127/how-to-bind-aspxscheduler-to-the-oracle-data-base-using-the-objectdatasource-component).

Reviewer: ```suggestion
For implementation details, refer to the following article: [How to bind ASPxScheduler to the Oracle database using the ObjectDataSource component (description)](https://supportcenter.devexpress.com/ticket/details/t223127/how-to-bind-aspxscheduler-to-the-oracle-data-base-using-the-objectdatasource-component).
```

> [!NOTE]

> > To run this example, you need to have an Oracle client installed on your machine and change connection string settings to the data base specified in the **Web.config** file.

Reviewer: ```suggestion
> To run this example, install an Oracle client on your machine and change the database connection string settings specified in the **Web.config** file.
```
